server/sonar-server/src/main/java/org/sonar/server/es/BaseIndexer.java

```diff
@@ -98,7 +100,9 @@ public BaseIndexer setEnabled(boolean b) {
 
   @Override
   public void start() {
-    lastUpdatedAt = esClient.getMaxFieldValue(indexName, typeName, dateFieldName);
+    if (!settings.getBoolean("sonar.internal.es.disableIndexes")) {
```

Comment:
Instead of using this property here, you should use the property 'enabled' -> 
if (enabled) {
   ...
}